scrapper/service/job_filter.py

```
@@ -0,0 +1,224 @@
+            extra={
+                "count": len(sorted_jobs),
+                "top_score": sorted_jobs[0].score if sorted_jobs else 0,
+                "min_score": sorted_jobs[-1].score if sorted_jobs else 0
+            }
+        )
+        
+        # Stage 5: Top-K Selection (apply limit)
+        if limit:
+            top_jobs = sorted_jobs[:limit]
+        else:
+            top_jobs = sorted_jobs
```

Ruff (0.15.12)

[warning] Use ternary operator `top_jobs = sorted_jobs[:limit] if limit else sorted_jobs` instead of `if`-`else`-block

Replace `if`-`else`-block with `top_jobs = sorted_jobs[:limit] if limit else sorted_jobs`

(SIM108)